### PR TITLE
Add CentOS 7 Support

### PR DESCRIPTION
**Pull request by kj4ezj**
As part of our ongoing [Travis CI beta testing](https://github.com/EOSIO/eos-vm/pull/38), we have added CentOS support to the eos-vm [Buildkite](https://buildkite.com/EOSIO/eos-vm-beta) and [Travis](https://travis-ci.org/EOSIO/eos-vm/builds) pipelines.   
  
As discussed in the Build Bros meeting on Tuesday, I have also broken the Docker entrypoint (actually `CMD`) and the `clang.make` cmake toolchain file into independent files, rather than constructing them in the Dockerfile. I believe this is cleaner, more readable, and less error prone.  
  
## Tested
- [eos-vm-base-images-beta build 220](https://buildkite.com/EOSIO/eos-vm-base-images-beta/builds/220#06ab3889-0052-4afb-b84f-332fcb0fd986) -- image built  
- [eos-vm-base-images-beta build 233](https://buildkite.com/EOSIO/eos-vm-base-images-beta/builds/233) -- image already exists  
- [Travis build 307](https://travis-ci.org/EOSIO/eos-vm/builds/566513040)  
- [eos-vm-beta build 209](https://buildkite.com/EOSIO/eos-vm-beta/builds/209)    
  
Thank you [scottarnette](https://github.com/scottarnette) for your help debugging for me on the [eos-vm:trav-poc-c7](https://github.com/EOSIO/eos-vm/commits/trav-poc-c7) branch.

### CODE REVIEW

File: .cicd/entrypoint.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```
Comment (scottarnette) [inline .cicd/entrypoint.sh]: Did we decide to use an bash script for our build CMD? If so, we should go ahead and port this across the other Dockerfiles in a later PR. Just something to note.

**Comment by scottarnette** [comment]
If this is building correctly in CentOS without LLVM, we should also removed LLVM from the other distros in a future PR.

File: .cicd/entrypoint.sh (hunk 1 above)
Comment (kj4ezj) [reply .cicd/entrypoint.sh]: This was something we discussed in the meeting on Tuesday (see PR description, paragraph 2).    
   
I should have done this in a separate PR, but I was bored waiting for builds so I added commit [`cc14159fe661e77bb425357d3f7b4750251b8b78`](https://github.com/EOSIO/eos-vm/pull/45/commits/cc14159fe661e77bb425357d3f7b4750251b8b78) and commit [`3c849fb3ab037f95eff51d5a053151699ef060c0`](https://github.com/EOSIO/eos-vm/pull/45/commits/3c849fb3ab037f95eff51d5a053151699ef060c0). They are both granular so we can revert them if that's what the group decides.

Comment (scottarnette) [reply .cicd/entrypoint.sh]: Gotcha. I'll approve and we can discuss further later.

File: .cicd/execute.sh [hunk 2]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+function execute()
+{
+  echo "$ $@"
```
[the remark this answers is not in the document]
Comment (kj4ezj) [reply .cicd/execute.sh]: I replaced code folding and cold foding.

**Pull request merged**